### DISCUSSION TITLE
Add support for area lights.

### DISCUSSION THREAD

**Discussion by catboy-catfish** [godotengine/godot-proposals#9761] [3D]
One of the features that Godot is missing right now is area lights. "Angular Distance" on OmniLight3Ds and SpotLight3Ds mention them. "This can simulate area lights to an extent."

I would appreciate if you could add that in for Godot Engine 4.4.

**Comment by AThousandShips** [comment]
Thank you for your proposal, consolidating in:
* https://github.com/godotengine/godot-proposals/issues/1275
